test/server/cards/01_SOR/leaders/IG88RuthlessBountyHunter.spec.ts

```diff
@@ -0,0 +1,86 @@
+describe('IG-88, Ruthless Bounty Hunter', function () {
+    integration(function (contextRef) {
+        describe('IG-88\'s leader undeployed ability', function () {
+            beforeEach(function () {
+                contextRef.setupTest({
+                    phase: 'action',
+                    player1: {
+                        groundArena: ['death-star-stormtrooper', 'battlefield-marine'],
+                        leader: 'ig88#ruthless-bounty-hunter',
+                        resources: 4,
+                    },
+                    player2: {
+                        groundArena: ['viper-probe-droid', 'wampa'],
+                    },
+                });
+            });
+
+            it('should attack with a friendly unit (without buff)', function () {
```

Comment:
nit: we should have a little more context on how the buff works - e.g., `should attack with a friendly unit and does not buff because there are fewer friendly units than enemy units`. Similar change to the other test below. 

Ideally the goal for test naming is that if the test breaks, someone could come here and read it and understand why the test is set up the way that it is without needing to look at the card